code/modules/client/preference_setup/loadout/loadout_accessories.dm

```diff
@@ -256,6 +294,142 @@
 	path = /obj/item/clothing/accessory/badge/idbadge/intel
 
 /datum/gear/accessory/sleeve_patch
-	display_name = "shoulder sleeve patch"
+	display_name = "shoulder sleeve patch (colourable)"
 	path = /obj/item/clothing/accessory/sleevepatch
 	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
+
+/datum/gear/accessory/namepin
+	display_name = "pin tag (colourable)"
+	path = /obj/item/clothing/accessory/badge/namepin
+	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
+
+/datum/gear/accessory/asym
+	display_name = "asymmetric jacket (selection)"
+	path = /obj/item/clothing/accessory/asymmetric
+
+/datum/gear/accessory/asym/New()
+	..()
+	var/asyms = list()
+	asyms["blue asymmetric jacket"] = /obj/item/clothing/accessory/asymmetric
+	asyms["purple asymmetric jacket"] = /obj/item/clothing/accessory/asymmetric/purple
+	asyms["green asymmetric jacket"] = /obj/item/clothing/accessory/asymmetric/green
+	gear_tweaks += new/datum/gear_tweak/path(asyms)
+
+/datum/gear/accessory/sash
```

Comment:
No sash please. We removed cloaks a while ago due to their conflict with the station and the setting's aesthetics.

Reply:
I'll consider removing them, but I also wish to understand what you believe is the setting's aesthetics and whether that was a determined decision in the design across multiple people rather than what could potentially just be your own vision—particularly when we're talking about the context of multiple players with effectively their own creative input in at least some form of agency with the backdrop of a very large (an understatement) section of the galaxy; even MORE SO when we're talking about entire planets that are prone to, no offence, caricatured versions of other real-life countries.
Now I might be uninformed, ignorant, "dramatic person" by virtue of not being in any lore team here, nor do I hold any particular power or responsibility in this community-based game, but that goes both ways in the sense that I do need help understanding what you guys talk about.

Reply:
There was a pr and a forum feedback topic about it, you can check them here: https://github.com/Aurorastation/Aurora.3/pull/5504

Reply:
That seems like a rather controversial topic that most people I'm seeing (who participated in the discussion, at least, never mind the entire server population) don't seem to agree with, although I do understand that this is the decision that was made regardless.
I'm **still open to the idea of removing it and I'll probably do so**, but I also still perceive this sort of double standard which I'm hoping you could help clear up where something that's considered a cultural piece of fashion is now considered ridiculous while we still keep or even add other things that effectively come down to just being Space Caucuses Region and Space 60s South Vietnam, as cool as I think they can be. It's the removal of something that existed for cultural purposes (I should point out that there are cultures in real life that incorporate cloaks and capes—to say they won't exist seems a bit of a folly to me when quite a lot of other lore is based upon quite a few cultures one would probably think won't exist by 2462 for                  whatever reasons) despite keeping other things for cultural purposes.
Now I'm not saying those should be removed and I'm not trying to draw attention to them. I just currently fail to understand where the apparent standard is here. I should point out that there are plenty of businesses I can give you, being from someone who lives in a city or even geographic region which has always been built upon international trade and commerce, in addition to tourism; cultural backlash is just not good for business unless you have a really specific setup in mind and even still that's a big risk. We are talking about a megacorporation that hires literally everyone they can across the Orion Arm, and to not build their brand with those things incorporated does seem like a missed opportunity, in-universe and player-wise.

But I digress: I still am not sure how sashes apply to this. The icon is a coloured band worn diagonally across the torso, not the types of sashes you wear... any other way that you may or may not be thinking.

Reply:
For reference, this is what the sash looks like (coloured in with relatively arbitrary colours):
<details>
<summary>Click</summary>
<img src="https://user-images.githubusercontent.com/3950115/85943580-c7f58780-b8e5-11ea-8846-4cdb34a3845d.png"><br>

Reply:
The issue was not only department's cloaks. If any piece in loadout is meant to represent something in lore, it should made clear what it is representing, regular clothing is not the case. About those factions; if anyone has access to the same stuff they have, they might as well don't have it, because whatever they are meant to represent is lost if they are just any regular piece of clothing. If anyone has everything and the aesthetic is all over the place; no one has anything in the end of the day.

Reply:
In that case, what if I move them under Xenowear- Human to give people second thoughts? There are practical uses to cloaks and I imagine not everyone's going to find 20°C  particularly temperate—especially considering the sort of cultures found in lore which seem heavily based on civilisations that have made heavy use of cloaks.

Reply:
No, they don't represent anything or any faction in the human lore, unlike the dominia and himeo clothing. Just throwing them into some faction is not the way to go. Sashes also fall under the very regal/royal clothing like capes and the like.